### DISCUSSION TITLE
Add markdown mime type in vweb

### DISCUSSION THREAD

**Discussion by timurgordon** [vlang/v#18688] [Ideas]
### Describe the feature

Add markdown mime type in vweb as .md and 'text/markdown'. This was feature was created and closed earlier in #5641. I'm not sure if the mime type was removed accidentally or intentionally. 

### Use Case

Serving files such as README.md

### Proposed Solution

add new mime type constant to mime types in vweb

### Acknowledgements

- [X] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### Version used

V 0.3.4 4485f1c

### Environment details (OS name and version, etc.)

OS: macos, macOS, 13.2.1, 22D68
Processor: 8 cpus, 64bit, little endian, Intel(R) Core(TM) i5-8259U CPU @ 2.30GHz

**Comment by jonathanouwerx** [comment]
Support for .md files was removed in [issue #11226](https://github.com/vlang/v/pull/11226), seemingly accidentally. So there shouldn't be any issues associated with it.
